vnext/ReactWindowsCore/WinRTWebSocketResource.cpp

```diff
@@ -354,6 +391,40 @@ void WinRTWebSocketResource::Synchronize()
   }
 }
 
+string WinRTWebSocketResource::GetRestrictedErrorMessage() noexcept
```

Comment:
we need more than a message, we need a stack trace to enable debugging

Reply:
Are you suggesting we publish the whole stack trace as part of the error message back to the React instance?

Or, that this method should return both the stack trace and the error message for debugging purposes?
If so, do you know how to retrieve the stack trace? (I'll look into the docs meanwhile...)

Reply:
I dont think RN devs are ever going to need the native callstack of a websocket failure.  If windows provided a websocket API that just returned an error message we'd use that, and we'd never get a callstack.

Websocket failures are expected, and the code handles them.  -- I don't think we are losing anything valuable by losing the callstack.